packages/@okta/vuepress-site/docs/guides/protect-your-api/main/springboot/configcors.md

```diff
@@ -1,12 +1,15 @@
-1. To configure CORS in Spring Security, enable it in the `WebSecurityConfigurerAdapter` that you defined in the previous step:
+1. To configure CORS in Spring Security, enable it in the `SecurityFilterChain` that you defined in the previous step:
 
    ```java
-   @Override
-   protected void configure(HttpSecurity http) throws Exception {
-     // previous configuration
-       ...
-     http.cors();
-   }
+    @Bean
+    SecurityFilterChain oauth2SecurityFilterChain2(HttpSecurity http) throws Exception {
```

Comment:
```suggestion
    SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
```

Simplify name as shown in https://spring.io/blog/2022/02/21/spring-security-without-the-websecurityconfigureradapter.